src/decorators/autobind.ts

```diff
@@ -1,18 +1,17 @@
-namespace App{
-    //autobind decorator
- export function autobind(
-    _target: any,
-    _methodName: string,
-    descriptor: PropertyDescriptor
-  ) {
-    const originalMethod = descriptor.value;
-    const adjDescriptor: PropertyDescriptor = {
-      configurable: true,
-      get() {
-        const boundFn = originalMethod.bind(this);
-        return boundFn;
-      },
-    };
-    return adjDescriptor;
-  }
-}
\ No newline at end of file
+//autobind decorator
+//decorator to automatically bind the this keyword method to the object instance
+export function autobind(
```

Comment:
do you understand what this does? haven't seen such kind of binding. Think about if you can do this in another way. Using arrow functions to register event handlers is usually a much cleaner way to handle this.

Reply:
If not used remove ( or at least comment ) this code